app/components/MapAreaControls.tsx

```diff
@@ -185,6 +187,31 @@ function MapAreaControls() {
           {selectionMode.type === "Selecting" ? selectionMode.name : "AOI"}
         </Box>
       )}
+      {validationError && (
+        <Box
+          px={3}
+          py={2}
+          bg="red.50"
+          borderColor="red.200"
+          borderWidth="1px"
+          borderRadius="md"
+          boxShadow="sm"
+          color="red.700"
+          fontSize="sm"
+        >
+          <Box fontWeight="bold" mb={1}>
+            {validationError.code === "too-small"
+              ? "Area is too small"
+              : "Area is too large"}
+          </Box>
+          <Box fontSize="xs">
+            Your area: {validationError.area.toLocaleString()} km²
```

Comment:
```suggestion
            Your area: {validationError.area.toLocaleString().toFixed(2)} km²
```

@vgeorge can we limit the decimals here?